blocks/serialization.py

```diff
@@ -544,13 +551,24 @@ def __call__(self, parameter):
         self.used_names.add(name)
         return name
 
+def _recreate_numpy_ndarray(_, content):
```

Comment:
To keep back-ward compatibility, you should swap those and maybe give a default value to _